core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -123,6 +125,27 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         </md-card>
       </div>
 
+      <div ng-if="explorationsList.length > 0" class="oppia-dashboard-tab-panel" style="padding-top: 13px;">
+        <span class="oppia-dashboard-total-elements">
+          You have a total of <[explorationsList.length]> explorations.
```

Comment:
(a) Does this need to be translated?
(b) This reads wrongly if the length is 1.
(c) Would it be cleaner to just include this as a number in parentheses in the corresponding tab -- e.g. "Explorations (1)"?